js/components/frame.js

```diff
@@ -334,6 +334,14 @@ class Frame extends ImmutableComponent {
         }
         this.webview.reloadIgnoringCache()
         break
+      case 'clone':
+        if (this.isAboutPage()) {
```

Comment:
Pls also add a similar check on the context menu to not even show the option to avid people thinking it's a bug.
Is it possible to use appUrlUtil's `isNavigatableAboutPage` here and there instead?